modules/testkit/src/test/scala/com/caesars/tracing/ZTracerInstrumentationSpec.scala

```diff
@@ -1,129 +1,124 @@
 package com.caesars.tracing
 
-import com.caesars.tracing.TracedBackend.TracedBackendConfig
 import com.caesars.tracing.TracingUtils.entryPointLayer
-import com.caesars.tracing.testing.*
-import io.janstenpickle.trace4cats.ToHeaders
-import io.janstenpickle.trace4cats.model.{SpanKind, SpanStatus}
-import org.typelevel.ci.CIString
 import sttp.client3.asynchttpclient.zio.AsyncHttpClientZioBackend
-import sttp.model.Header
 import zhttp.service.EventLoopGroup
 import zhttp.service.server.ServerChannelFactory
 import zio.*
-import zio.magic.*
-import zio.test.*
+import zio.test.{DefaultRunnableSpec, assertTrue, assert, assertM}
```

Comment:
minor, but can you run scalafmt here to get these imports alphabetized properly?